src/main/java/com/gluonhq/substrate/target/TripletProfile.java

```diff
@@ -0,0 +1,42 @@
+package com.gluonhq.substrate.target;
+
+import com.gluonhq.substrate.Constants;
+
+/**
+ * Predefined Profiles
+ */
+public enum TripletProfile {
```

Comment:
This enum is actually the os-vendor combination of the triplet. I don't think we can map it like that. For example, Linux on AArch64 needs a separate profile (aarch64-linux-linux)